### PR TITLE
Reliable UI

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

Ensures all data needed are loaded before values are calculated and shown to user for...

- Profit range (Norm Factor needed to be loaded)
- collateral ratio (Norm Factor needed to be loaded)
- withdrawal amount  (Squeeth Initial price needed to be loaded)

Fixes ENG-842

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

**Review by KMKoushik — APPROVED** [review]

File: packages/frontend/pages/strategies.tsx [hunk 1]
```diff
@@ -352,25 +355,54 @@ const Strategies: React.FC = () => {
...
-                        'Approx Profitable (' + (profitableMovePercentV2 * 100).toFixed(2) + '%) between ETH Prices ($)'
-                      }
-                      tooltip={Tooltips.StrategyProfitThreshold}
-                    />
+                    
+                    crabLoading ? (
+                      <>
+                          <StrategyInfoItem
+                          value='-'
+                          label={
```
Comment (sanandnarayan) [inline packages/frontend/pages/strategies.tsx:363]: @DemolaJames can you bring the if condition inside label .

i.e const lowerBand = crabLoading ? "-" : lowerPriceBandForProfitability.toFixed(2)

File: packages/frontend/pages/strategies.tsx [hunk 2]
```diff
@@ -352,25 +355,54 @@ const Strategies: React.FC = () => {
...
+                        }
+                        label={
+                          'Approx Profitable (' + (profitableMovePercentV2 * 100).toFixed(2) + '%) between ETH Prices ($)'
+                        }
+                        tooltip={Tooltips.StrategyProfitThreshold}
+                      />
+
+                      <StrategyInfoItem
+                      link="https://squeeth.opyn.co/vault/286"
+                      value={collatRatio === Infinity ? '0.00' : collatRatio.toString()}
```
Comment (sanandnarayan) [inline packages/frontend/pages/strategies.tsx:393]: Instead of 0.0 can we make it - ?
Is it ever Infinity?

**Review by sanandnarayan — COMMENTED** [review]
needs some changes

File: packages/frontend/pages/strategies.tsx (hunk 1 above)
Comment (DemolaJames) [reply packages/frontend/pages/strategies.tsx]: Refactored

File: packages/frontend/pages/strategies.tsx (hunk 2 above)
Comment (DemolaJames) [reply packages/frontend/pages/strategies.tsx]: Updated to read '-'
Its only Infinity when the contract information is not loaded.

**Pull request merged**